LanguageFeatures/Inline-classes/static_analysis_inline_class_A04_t01.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A compile-time error occurs if a type parameter of an inline
+/// class occurs in a non-covariant position in the representation type.
+///
+/// @description Checks that it is a compile-time error if a type parameter of
+/// an inline class occurs in a non-covariant position in the representation
+/// type
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=inline-class
+
+class A {}
+class B extends A {}
+class C extends B {}
+
+inline class V<T extends B> {
+  final T id;
```

Comment:
That won't work: The type annotation of a final instance variable is a covariant position.

But `final void Function(T) f;` would work.

Reply:
(I wrote a comment in the overall review entry: We should not forget about this topic)